src/compiler/core/src/commonTest/kotlin/community/flock/wirespec/compiler/core/CompileFullEndpointTest.kt

```diff
@@ -56,38 +57,40 @@ class CompileFullEndpointTest {
             |
             |  data class Queries(
             |    val done: Boolean,
+            |    val name: String?,
             |  ) : Wirespec.Queries
             |
             |  data class Headers(
             |    val token: Token,
+            |    val refreshToken: Token?,
             |  ) : Wirespec.Request.Headers
             |
             |  class Request(
             |    id: String,
-            |    done: Boolean,
-            |    token: Token,
+            |    done: Boolean,     name: String?,
+            |    token: Token,     refreshToken: Token?,
```

Comment:
Nitpick, most probably, we need to add newlines in the kotlin emitter (out of scope for this MR)